miscellaneous/TopGG.py

```diff
@@ -13,7 +13,7 @@ class TopGG(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
         self.token = config['APIs']['top.gg']
-        self.dblpy = topgg.DBLClient(self.bot, self.token, autopost=True, post_shard_count=True)
+        # self.dblpy = topgg.DBLClient(self.bot, self.token, autopost=True, post_shard_count=True)
```

Comment:
Make sure to uncomment this

Reply:
Oh man, sorry. Didn't realise I added that file too for that commit.